### PR TITLE
Add an initial GitHub Actions configuration

### PR DESCRIPTION
**Pull request by LujieZ**

### CODE REVIEW

**Comment by LujieZ** [comment]
@googlebot I fixed it. The commit was pushing through my gmail account `lujiesue@gmail.com`, and I changed the user.email config to my google account `lujiez@google.com`.

**Comment by LujieZ** [comment]
@googlebot I fixed it. Recreated the previous commit!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v1
+
+      - name: Use Node 12.x
```
Comment (razvanm) [inline .github/workflows/main.yml:17]: How come `actions/setup-node@v1` means Node 12.x?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: Build Angular on GitHub Actions
+
+on:
+  push:
+    branches: 
+      - github-actions
+
+jobs:
+  build:
+    name: Build and Test
```
Comment (razvanm) [inline .github/workflows/main.yml:10]:
```suggestion
    name: Lint, Test, E2E
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    steps:
+      - name: Check out
+        uses: actions/checkout@v1
+
+      - name: Use Node 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version:
+            ${{ matrix.node-version }}
```
Comment (razvanm) [inline .github/workflows/main.yml:24]: Do we need to use a matrix? We only care about a single version now.

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    branches: 
+      - github-actions
+
+jobs:
+  build:
+    name: Lint, Test, E2E
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (razvanm) [inline .github/workflows/main.yml:14]: One more thing: we use 10.21 on our remote environment so we should use for the CI configuration too.

**Pull request merged**